src/main/utils.ts

```diff
@@ -29,6 +29,18 @@ import * as path from 'path';
 import * as fs from 'fs';
 import log from 'electron-log';
 
+export
+interface IAppConfiguration {
+    jlabPort: number;
+    token: string;
+};
+
+export
+const appConfig: IAppConfiguration = {
+    jlabPort: 8888,
+    token: 'jlab-token'
+};
```

Comment:
I anticipate that a hard-coded global will cause issues once we will go for full front/back separation. But I am happy to address it myself later.

Reply:
yes, we can definitely improve this when we support remote backends etc. by the way, these are just default values and token is generated on the fly before server launch.